models/users.js

```diff
@@ -58,10 +58,17 @@ const addOrUpdate = async (userData, userId = null) => {
  */
 const fetchUsers = async (query) => {
   try {
-    const snapshot = await userModel
+    let snapshot = await userModel
       .limit(parseInt(query.size) || 100)
       .offset((parseInt(query.size) || 100) * (parseInt(query.page) || 0))
       .get();
+    if (query.role === "members") {
```

Comment:
@aakankshadhurandhar Could we move the `members` string into an enum or a constants file please?

Reply:
@ankushdharkar  I have imported it directly now from [roles.js](https://github.com/Real-Dev-Squad/website-backend/blob/develop/constants/roles.js)